content/docs/introducing-jsx.md

```diff
@@ -70,13 +71,13 @@ ReactDOM.render(
 
 [](codepen://introducing-jsx)
 
-We split JSX over multiple lines for readability. While it isn't required, when doing this, we also recommend wrapping it in parentheses to avoid the pitfalls of [automatic semicolon insertion](http://stackoverflow.com/q/2846283).
+Separamos o JSX em múltiplas linhas para melhorar a legibilidade. Mesmo que não seja requerido, quando fizer isso, também recomendamos cercá-lo com parênteses para evitar as armadilhas da [inserção automática de ponto-e-vírgula](http://stackoverflow.com/q/2846283).
 
-### JSX is an Expression Too {#jsx-is-an-expression-too}
+### JSX Também é uma Expressão {#jsx-is-an-expression-too}
 
-After compilation, JSX expressions become regular JavaScript function calls and evaluate to JavaScript objects.
+Depois da compilação, as expressões em JSX se transformam em chamadas normais de funções em JavaScript e são avaliadas para objetos JavaScript.
```

Comment:
```suggestion
Depois da compilação, as expressões em JSX se transformam em chamadas normais de funções que retornam objetos JavaScript.
```

Reply:
Here the term _"evaluate"_ means more as a return of a function other than _"avaliar"_.

Reply:
True!